src/components/Signup/Signup.js

```diff
@@ -99,27 +100,7 @@ export default injectIntl(function({ intl }) {
       <article className="main-panel">
         <header>
           <h1 className="logo-doppler-new">Doppler</h1>
-          <div className="language-selector">
-            {intl.locale === 'es' ? (
-              <div className="option" id="spanish-selector">
-                <span>
-                  <strong>ES</strong>
-                </span>
-                <Link to="?lang=en" className="option--item">
-                  EN
-                </Link>
-              </div>
-            ) : (
-              <div className="option" id="spanish-selector">
-                <span>
-                  <strong>EN</strong>
-                </span>
-                <Link to="?lang=es" className="option--item">
-                  ES
-                </Link>
-              </div>
-            )}
-          </div>
+          <LanguageSelector lang={intl.locale} />
```

Comment:
I am not sure, is just an opinion, what do you think calling it `currentLang`?

Reply:
I like that approach, @andresmoschini . Will do the proper changes 👍